packages/terra-table/src/TableSubheader.jsx

```diff
@@ -19,21 +22,14 @@ const TableSubheader = ({
   content,
   colSpan,
   ...customProps
-}) => {
-  const contentClassName = classNames([
-    'terra-Table-subheader',
-    customProps.className,
-  ]);
-
+}) => (
   // count is based on the number of columns and assigned in the table component which contains this subheader
-  return (
-    <tr className="terra-Table-subheaderRow">
-      <td {...customProps} className={contentClassName} colSpan={colSpan}>
-        {content}
-      </td>
-    </tr>
-  );
-};
+  <tr data-class="subheader-row">
+    <td {...customProps} className={cx('subheader', customProps.className)} colSpan={colSpan}>
```

Comment:
On [line 23](https://github.com/cerner/terra-core/pull/656/files/8934dc09cbc70d876252a6bbc8ca26d7a0e09e67#diff-b21a6067f2e7eca6ed91fa3729b329f3L23), `customProps.className` was included. I moved the className rendering to be inline, as it has been a suggestion on other PR by @mhemesath.